src/semantic.less

```diff
@@ -13,63 +13,167 @@
 // stylelint-disable no-invalid-position-at-import-rule
 
 /* Global */
-& { @import 'definitions/globals/reset'; }
-& { @import 'definitions/globals/site'; }
+& {
```

Comment:
Can we ignore the forced extra lines for semantic.less? The idea behind this file is to integrate into an existing less project and having each component in a single line would make it easiert to quickly remove/disable a component.
"Possibly" people could have some own CI which is adjusting this fileand are expecting single lines and would break then.

Reply:
```diff
 & {
-    @import(...);
+//  @import(...);
 }
```

this should be fine

Reply:
Yes sure, but we actually dont know what others are doing so it might be still a braking change.
_Please_ disable the rule for this file. Thanks.

Reply:
here fixed and added https://github.com/fomantic/Fomantic-UI/pull/2611/files#r1050733837 comment to the Prettier PR to make sure it stays as wanted